Lib/pathlib.py

```diff
@@ -1069,15 +1067,10 @@ def _make_child_relpath(self, part):
         return self._from_parsed_parts(self._drv, self._root, parts)
 
     def __enter__(self):
```

Comment:
I would add a comment referring to the issue that led making the context manager support a no-op, what version it became a no-op, and a comment that it can probably eventually be removed after a reasonable amount of time.